notebooks/quickstart/data-owner/01-upload-data.ipynb

```diff
@@ -215,7 +215,7 @@
    "outputs": [],
    "source": [
     "# run this cell\n",
-    "data_subjects = DataSubjectList.from_series(dataset[\"ID\"])\n",
+    "data_subjects = DataSubjectArray.from_series(dataset[\"ID\"])\n",
```

Comment:
Hi Mrinal!

This does not work because DataSubjectArray is still not defined anywhere. 

You can do
'sy.DataSubjectArray.from_objs(dataset["ID"])'

Please run the notebook whenever you make a change to make sure it works.

Thanks!